invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js

```diff
@@ -4,41 +4,40 @@
 // InvenioRDM is free software; you can redistribute it and/or modify it
 // under the terms of the MIT License; see LICENSE file for more details.
 
-import _truncate from "lodash/truncate";
-import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
 import React from "react";
-import { CommunityTypeLabel } from "../labels";
+import PropTypes from "prop-types";
+import _truncate from "lodash/truncate";
 
-import { Grid, Container, Image, Item } from "semantic-ui-react";
-import { RestrictedLabel } from "../labels";
+import { Image, InvenioPopup } from "react-invenio-forms";
+import { Item, Grid, Icon } from "semantic-ui-react";
+import { CommunityTypeLabel, RestrictedLabel } from "../labels";
 
 export const CommunityCompactItemComputer = ({
   result,
   actions,
   extraLabels,
   itemClassName,
+  showPermissionLabel,
 }) => {
-  const { metadata, links, access, id } = result;
-  const ui = result.ui;
-
+  const { metadata, ui, links, access, id } = result;
   const communityType = ui?.type?.title_l10n;
 
   return (
     <Item
       key={id}
       className={`computer tablet only justify-space-between community-item ${itemClassName}`}
     >
-      <Image as={Item.Image} size="tiny" src={links.logo} />
+      <Image size="tiny" src={links.logo} alt="" />
```

Comment:
```suggestion
      <Image size="tiny" src={links.logo} alt={i18next.t("Community logo")} />
```

Reply:
I don't think this is needed, as the logo doesn't have any functionality and doesn't provide any information, ref [https://www.w3.org/WAI/tutorials/images/](https://www.w3.org/WAI/tutorials/images/)